index.html

```diff
@@ -0,0 +1,192 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Portfolio setup</title>
+  <link rel="stylesheet" href="./style.css">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Poppins:wght@300;400;500&display=swap" rel="stylesheet">
+<link href="https://fonts.googleapis.com/css2?family=Playfair+Display&display=swap" rel="stylesheet">
+<link href="https://fonts.googleapis.com/css2?family=Poppins:wght@600;800&display=swap" rel="stylesheet">
+<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+</head>
+<body>
+  <header>
+    <div class="logo">
+      <img  src="./images/image_geometry_1.png" alt="logo image">
+   </div>
+    <nav>
+      <div class="menu">
+           <i class="fa fa-bars"></i>
+      </div>
+    </nav>
+  </header>
+  <main>
+  
+    <section class="headline"> 
+        <h1 class="myname">Charlie 
+        Giorgania
+        </h1> 
+        <p  class="headline-p">Hello! I am a software developer! I can help you build a product, feature or website. Take a look at my work. <br> <br>
+  
+          If you like what you see and have a project you need coded, don’t hesitate to contact me.
+        </p>
+      <button class="headline-btn">
+        Start collaboration
+      </button>
+      <div class="headline-img">
+        <img src="./images/image_geometry_2.png" alt="intro image">
+      </div>
+  </section>
+ <section class="projects">
+          <h2 class="project-title">Projects</h2>
+       <div class="projects-card">
```

Comment:
- [ ] As suggested by the previous reviewer and demanded by project requirements, kindly align the contents of the project cards container using CSS grid.

- [Optional] I see that you named your cards container `projects-card`, also you are splitting each individual card into divs. Please note that a card is representing a single project as shown below and should be in the same container.
![image](https://github.com/hydaline-code/Portfolio_mobilefirst/assets/94551484/e5fe1378-7edd-4c53-8851-9a28ff937ca8)